### PR TITLE
feat: output html

### LINKED ISSUE #20: export result to html
New `format` in `venom run` command: `html`

Generate a pretty html report to navigate in all results, with emojis(💩), charts, graphs, and so on...

### PR DESCRIPTION
**Pull request by yesnault**
example of usage:
venom run --format=html --output-dir=. *.yml
venom run --format=html,json --output-dir=. *.yml

refact internal struct, json output is more detailed

close #20

Signed-off-by: Yvonnick Esnault <yvonnick.esnault@corp.ovh.com>

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#980.0 ✘  
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ✘

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#982.0 ✘  
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ✘

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#984.0 ■   
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ■

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#984.0 ■   
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ■

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#985.0 ✘  
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ✘

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#987.0 ✘  
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ✘

File: process_testcase.go [hunk 1]
```diff
@@ -176,16 +179,21 @@ func (v *Venom) runTestSteps(ctx context.Context, tc *TestCase, tsIn *TestStepRe
 		stepVars.AddAllWithPrefix(tc.Name, tc.computedVars)
 		stepVars.Add("venom.teststep.number", stepNumber)
 
+		//testStepResult := TestStepResult{}
```
Comment (fsamin) [inline process_testcase.go:182]: here

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#994.0 ✘  
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ✘

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#996.0 ✘  
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ✘

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#997.0 ✘  
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ✘

**Review by fsamin — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#998.0 ✘  
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ✘

**Pull request merged**